homeassistant/components/lacrosse_view/coordinator.py

```diff
@@ -60,24 +61,34 @@ async def _async_update_data(self) -> list[Sensor]:
             except LoginError as error:
                 raise ConfigEntryAuthFailed from error
 
+        if self.devices is None:
+            _LOGGER.debug("Getting devices")
+            try:
+                self.devices = await self.api.get_devices(
+                    location=Location(id=self.id, name=self.name),
+                )
+            except HTTPError as error:
+                raise ConfigEntryNotReady from error
```

Comment:
```suggestion
                raise UpdateFailed from error
```
during a normal update we raise UpdateFailed